roles/startmanager/vars/main.yml

```diff
@@ -15,7 +15,7 @@
 
 single_node: false
 
-manager_ip: "{{ ansible_host }}"
+manager_ip: 10.0.0.101
```

Comment:
@j0hnL Should this be a hardcoded IP?

Reply:
replaced with default ipv4 address from `facts`